SQL/0000-00-00-schema.sql

```diff
@@ -754,6 +759,98 @@ CREATE TABLE `mri_protocol_checks` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
 
+-- ********************************
+-- BIDS tables
+-- ********************************
+
+CREATE TABLE `bids_category` (
+ `BIDSCategoryID`  int(3)      NOT NULL AUTO_INCREMENT,
+ `ImagingCategory` varchar(10) NOT NULL UNIQUE,
+ PRIMARY KEY (`BIDSCategoryID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO `bids_category` (ImagingCategory) VALUES
+      ('anat'),
+      ('func'),
+      ('dwi'),
+      ('fmap');
+
+CREATE TABLE `bids_scan_type_subcategory` (
+  `BIDSScanTypeSubCategoryID` int(3)       NOT NULL AUTO_INCREMENT,
+  `BIDSScanTypeSubCategory`   varchar(100) NOT NULL UNIQUE,
+  PRIMARY KEY (`BIDSScanTypeSubCategoryID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO `bids_scan_type_subcategory` (BIDSScanTypeSubCategory) VALUES
+  ('task-rest');
+
+CREATE TABLE `bids_scan_type` (
+  `BIDSScanTypeID` int(3)       NOT NULL AUTO_INCREMENT,
+  `BIDSScanType`   varchar(100) NOT NULL UNIQUE,
+  PRIMARY KEY (`BIDSScanTypeID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO `bids_scan_type` (BIDSScanType) VALUES
+  ('bold'),
+  ('FLAIR'),
+  ('T1w'),
+  ('T2w'),
+  ('dwi');
+
+CREATE TABLE `bids_mri_scan_type_rel` (
+  `MRIScanTypeID`             int(10) UNSIGNED NOT NULL,
+  `BIDSCategoryID`            int(3)           DEFAULT NULL,
+  `BIDSScanTypeSubCategoryID` int(3)           DEFAULT NULL,
+  `BIDSScanTypeID`            int(3)           DEFAULT NULL,
+  `BIDSEchoNumber`            int(3)           DEFAULT NULL,
+  PRIMARY KEY  (`MRIScanTypeID`),
+  KEY `FK_bids_mri_scan_type_rel` (`MRIScanTypeID`),
+  CONSTRAINT `FK_bids_mri_scan_type_rel`     FOREIGN KEY (`MRIScanTypeID`)             REFERENCES `mri_scan_type` (`ID`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `FK_bids_category`              FOREIGN KEY (`BIDSCategoryID`)            REFERENCES `bids_category`(`BIDSCategoryID`),
+  CONSTRAINT `FK_bids_scan_type_subcategory` FOREIGN KEY (`BIDSScanTypeSubCategoryID`) REFERENCES `bids_scan_type_subcategory` (`BIDSScanTypeSubCategoryID`),
+  CONSTRAINT `FK_bids_scan_type`             FOREIGN KEY (`BIDSScanTypeID`)            REFERENCES `bids_scan_type` (`BIDSScanTypeID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- Default schema scan types; make some of them named in a BIDS compliant manner
+INSERT INTO bids_mri_scan_type_rel
+  (MRIScanTypeID, BIDSCategoryID, BIDSScanTypeSubCategoryID, BIDSScanTypeID, BIDSEchoNumber)
+  VALUES
+  (
+    (SELECT ID FROM mri_scan_type WHERE Scan_type = 'flair'),
+    (SELECT BIDSCategoryID FROM bids_category WHERE BIDSCategory='anat'),
```

Comment:
/BIDSCategory/ImagingCategory

and in all other values sets

Reply:
Good catch!! Should be fixed now.

Reply:
@PapillonMcGill check the very last commit for those changes. 

I squashed all the other commits into one so I can see what changed in the tables into 2 commits instead of many for the MRI code changes.